### PR TITLE
Merge main into refactor

### PR DESCRIPTION
**Pull request by jasondet**

### CODE REVIEW

**Comment by jasondet** [comment]
@gipert I'm trying to merge recent main changes into refactor but there were some conflicts in docs. I think I resolved them all but can you check them?

File: docs/source/conf.py [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 import pathlib
 import sys
-
+import os
```
Comment (gipert) [inline docs/source/conf.py:5]: This import is not needed

File: docs/source/index.rst [hunk 2]
```diff
@@ -18,3 +18,10 @@ Table of Contents
    tutorials
    Package API reference <generated/pygama>
    contributing
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
+* :ref:`search`
```
Comment (gipert) [inline docs/source/index.rst:27]: Remove these added lines please

**Review by gipert — CHANGES_REQUESTED** [review]

File: docs/source/conf.py (hunk 1 above)
Comment (jasondet) [reply docs/source/conf.py]: done

File: docs/source/index.rst (hunk 2 above)
Comment (jasondet) [reply docs/source/index.rst]: done

**Review by gipert — APPROVED** [review]

**Pull request merged**